GVFS/GVFS.Installer.Mac/scripts/postinstall

```diff
@@ -1,4 +1,15 @@
 #!/bin/bash
+
+KEXTBUNDLEID="org.vfsforgit.PrjFSKext"
+KEXTPATH="/Library/Extensions/PrjFSKext.kext"
+isKextLoadedCmd="/usr/sbin/kextstat -l -b $KEXTBUNDLEID | wc -l"
+isKextLoaded=$(eval $isKextLoadedCmd)
+if [ "$isKextLoaded" -eq 0 ]; then
+    loadCmd="/sbin/kextload \"$KEXTPATH\""
+    echo $loadCmd
```

Comment:
In other places, it looks like we include a little more context around the operation, rather than just the command. Should we do that here as well?

Reply:
Added a short comment with info on what we are trying to do.